lms/static/sass/course/_auto-cert.scss

```diff
@@ -0,0 +1,32 @@
+.wrapper-auto-cert {
+    display: table;
+    width: 100%;
+    clear: both;
+    margin: 0; // Overrides .wrapper-msg
+    padding: 0; // Overrides .wrapper-msg
+    background: transparent; // Overrides .wrapper-msg
```

Comment:
Set this to none instead of transparent, unless there's a good reason for transparent.